demo/pages/edit-templates/index.js

```diff
@@ -83,6 +83,43 @@ export default function Controller($http) {
 			key: 'memberSince',
 			title: 'Member Since',
 			type: 'date'
+		},
+		{
+			key: 'modifiedTime',
+			title: 'Modified Time',
+			type: 'time',
+			value: (item, value) => isUndef(value) ? item.modified || '' : item.modified = value
+		},
+		{
+			key: 'webPage',
+			title: 'Web Page',
+			type: 'url',
+			value: (item, value) => isUndef(value)
+				? item.webPage || `https://corp.portal.com/${item.name.last}.${item.name.first}`
+				: item.webPage = value,
+			label: (item, label) => isUndef(label)
+				? item.webPageLabel || `${item.name.last} ${item.name.first}`
+				: item.webPageLabel = label
+		},
+		{
+			key: 'attachment',
+			title: 'Attachment',
+			type: 'file',
+			value: (item, value) => isUndef(value) ? item.attachment : item.attachment = value,
+			onUpload: (e) => {
```

Comment:
Depends on use-case. If items are uploaded separately, it makes sense.
But i'd rather have `upload` instead of it, because only user knows how to upload file